### ISSUE TITLE
Did you train by yourself and test it?

### ISSUE THREAD

**Issue by kashiwachen** [hizhangp/yolo_tensorflow#62]
Hi, recently I train your net with my own dataset. However when I test with the training data, the result is nothing. I cannot understand it. So I want to know whether it works on your PC

**Comment by bailiqun** [comment]
met same problem with you..

**Comment by NineBall9** [comment]
the same problem. There is no errors while training and the test picture output nothing.

**Comment by lichacha** [comment]
same

**Comment by NineBall9** [comment]
Same! Does anyone can explain it.

**Comment by haixiaotian** [comment]
same

**Comment by Tshzzz** [comment]
maybe you didn't use pretrain file! see the config.py use weight

**Comment by lichacha** [comment]
I tested in VOC2007, testing yolosmall.ckpt directly is OK, training in voc2007 no output, and the use use pretrain file would be worse until there was no output.  Are there too few training times？  I did 20k  no output

**Comment by bailiqun** [comment]
have look with tensorflow/models/object_detection , may help

**Comment by lichacha** [comment]
Where is this file？  this project？

**Comment by contactmat85** [comment]
same. I training with a new data set, with just one class. I use transfer learning initializing only the convolutional layer with yolosmall.ckpt. I have a good a quick convergency but no output, on the training set neither
![train](https://user-images.githubusercontent.com/27950755/43900826-3a3fe2e6-9be6-11e8-98bf-0b96eb5d003f.png)

**Comment by wheatjo** [comment]
same,i have trained one class object, but output values are near zero

**Comment by YunYang1994** [comment]
hope this helps you ! https://github.com/YunYang1994/tensorflow-yolov3

**Comment by yanlongbinluck** [comment]
same problem to you

**Comment by Adienly** [comment]
SAME, my loss is about 2(use my own dataset, imagesize is 90, cellsize is 2), when i test, there is nothing

**Comment by kashiwachen** [comment]
I already finished the project taking your framework as a reference.
Happy develope!
